tests/test_distributions.py

```diff
@@ -262,6 +262,15 @@
             "peak": np.array([0.5, 0.5], dtype="float32"),
         },
     },
+    "TruncatedNormal": {
+        "scalar_parameters": {"loc": 0.0, "scale": 1.0, "low": 0.0, "high": 2.0},
+        "multidim_parameters": {
+            "loc": np.array([0.0, 0.0], dtype="float32"),
+            "scale": np.array([1.0, 1.0], dtype="float32"),
+            "low": np.array([0.0, 0.0]),
+            "high": np.array([2.0, 2.0]),
```

Comment:
Another thing to look is `common_dtype` method [here](https://github.com/tensorflow/probability/blob/v0.9.0/tensorflow_probability/python/distributions/truncated_normal.py#L143). You may run into compatibility issues between `np.float32` and `np.float64`.